python/pynessie/commands/_branch_tag_handlers.py

```diff
@@ -99,16 +101,33 @@ def _handle_json_output(input_data: list, ref_name: str) -> str:
     return ReferenceSchema().dumps(input_data, many=True)
 
 
-def _handle_normal_output(input_data: list, verbose: bool, default_branch: str) -> str:
+def _handle_normal_output(input_data: list, verbose: bool, default_branch: str, show_additional_info: bool = False) -> str:
     output = ""
     max_width = max((len(i.name) for i in input_data), default=0)
     for x in input_data:
+        additional_info = ""
+        if show_additional_info and x.metadata:
+            num_commits_ahead = x.metadata.num_commits_ahead if x.metadata.num_commits_ahead else "?"
+            num_commits_behind = x.metadata.num_commits_behind if x.metadata.num_commits_behind else "?"
+            num_total_commits = x.metadata.num_total_commits if x.metadata.num_total_commits else "?"
+            common_ancestor_hash = x.metadata.common_ancestor_hash if x.metadata.common_ancestor_hash else "?"
+            head = (
+                "[author: {} | commitTime: {} | msg: {}]".format(
+                    x.metadata.commit_meta_of_head.author, x.metadata.commit_meta_of_head.commitTime, x.metadata.commit_meta_of_head.message
+                )
+                if x.metadata.commit_meta_of_head
+                else "?"
+            )
+            additional_info = " [commits ahead: {} | commits behind: {} | total commits: {} | common ancestor: {} | HEAD: {}]".format(
```

Comment:
nit: perhaps use simply `total commits: {}, behind: {}, ahead: {}...` to make it terser in console output? Also putting `behind` first aligns with how GH renders those counts in the UI, I guess, where the "behind" count is on the left.

Reply:
rename `total commits` -> `commits from root` for clarity?

Reply:
updated to reflect your suggestion. regarding total commits -> commits from root rename, I wonder if that is really clearer for users. To me `total commits` is pretty clear, but `commits from root` is rather confusing

Reply:
maybe `commits on branch`? `total` sounds to me like the number of commits for the whole repo, but if you think it's ok, I'd be fine with `total` too.

Reply:
actually, for me personally the "number of commits from the common ancestor" would be interesting, but it's not for this PR for sure :)

Reply:
given that a user is looking at the metadata for a single branch/tag I think it should be self-explanatory that `total` means the total commits within this given branch/tag. Therefore I would go with this for now. We can rename it if we see that it's unclear to users what that really means.